.github/workflows/main.yml

```diff
@@ -31,6 +31,11 @@ jobs:
     secrets:
       GCP_CREDENTIALS: ${{ secrets.GCP_CREDENTIALS }}
 
+  npm_publish:
+    uses: viamrobotics/rdk/.github/workflows/npm-publish.yml@main
```

Comment:
Add  "needs: test" here, so this doesn't fire until/unless the tests pass.

Reply:
Good catch!